geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/serial/SerialGatewaySenderOperationsDistributedTest.java

```diff
@@ -618,20 +618,20 @@ public void test_Bug44153_StopOneSender_StartAnotherSender_CheckQueueSize() {
 
     vm5.invoke(() -> doPuts(className + "_RR", 10, 110));
 
-    vm5.invoke(() -> validateQueueContents("ln", 100));
+    vm5.invoke(() -> validateQueueContents("ln", 110));
     vm5.invoke(() -> stopSender("ln"));
     vm5.invoke(() -> validateSenderStoppedState("ln"));
 
     vm4.invoke(() -> startSender("ln"));
-    vm4.invoke(() -> validateQueueContents("ln", 10));
+    vm4.invoke(() -> validateQueueContents("ln", 110));
```

Comment:
Is this because we are no longer clearing the queue when stopping a gateway sender?

Reply:
Yes